features/cli/deploy.feature

```diff
@@ -377,8 +381,11 @@ Feature: deployment related features
       | name     | ab-example-a     |
       | replicas | 1                |
     Then the step should succeed
```

Comment:
@zhouying7780 Do we need this line ? I think we can remove it, what do you say ?

Reply:
@kasturinarra   the scale active is needed.

Reply:
@zhouying7780 okay, thanks !! Got it